spec/features/meta_keys_spec.rb

```diff
@@ -567,6 +567,9 @@ def meta_datum_types
         expect(current_path).to eq(meta_key_path(meta_key))
         expect(meta_key.meta_data.reload).to eq(meta_data)
 
+        visit meta_key_path("")
```

Comment:
does this work? normally one should pass the meta_key to this path helper.

Reply:
Yes, fnc only adds meta_key to url `/admin/meta_keys/ `
Both scenarios have been tested with binding.pry - with/out fix

Reply:
Replaced it by string-definition